lua/obsidian/ui.lua

```diff
@@ -515,14 +691,30 @@ local function update_extmarks(bufnr, ns_id, ui_opts)
 
     -- Check if inside a code block or at code block boundary. If not, update marks.
     if string.match(line, "^%s*```[^`]*$") then
-      inside_code_block = not inside_code_block
+      inside_code_block = true
       -- Remove any existing marks here on the boundary of a code block.
       clear_line()
     elseif not inside_code_block then
+      -- Mark that we have a highlight group now
+      if string.match(line, search.Patterns.Callout) then
+        local count = 0
+        for c = 1, #line do
+          local char = line:sub(c, c)
+          if char == ">" then
+            count = count + 1
+          elseif char == "[" then
+            if line:sub(c, c + 2) == "[!]" then
+              break
+            end
+          end
+        end
```

Comment:
If this is just counting the number of ">"s then you could do that with regular expressions

Reply:
I'm a n00b with Lua and regex so I'm struggling to find the correct way to do this. 
Do you have any tips?

Reply:
I think you could just do `local count = util.string_count(line, "> ")`